core/extensions.js

```diff
@@ -264,8 +253,19 @@ Blockly.Extensions.checkBlockHasMutatorProperties_ = function(errorPrefix,
  */
 Blockly.Extensions.getMutatorProperties_ = function(block) {
```

Comment:
Null is now an option again. I fixed this method to explicitly check for existence, not truthiness.